README.md

```diff
@@ -51,37 +70,39 @@ Execute a query:
+If a query is executed successfully, `0` will be returned. For a more complete example, see [Graph Java client example](./examples/src/main/java/com/vesoft/nebula/examples/GraphClientExample.java).
 
-### Storage Client
+### Storage client example
 
-If you only use the interface of RPC, nothing to worry about.
+If you only use the interface of RPC, nothing need to be done.
 
-If you want to directly use storage client to encode/decode, you need to use the jni interface. We have already package a `libnebula_codec.so` in the [nebula-utils](https://repo1.maven.org/maven2/com/vesoft/nebula-utils/) jar package.
+If you want to directly use the storage client to encode or decode, you must use the `jni` interface. We have already packaged `libnebula_codec.so` in the [nebula-utils](https://repo1.maven.org/maven2/com/vesoft/nebula-utils/) jar package.
 
-However, if it doesn't work in your environment, you can compile it by following steps:
-- Compile the [dynamic link library](https://github.com/vesoft-inc/nebula/tree/master/src/jni). 
-- Then install the jni jar into your local maven repo:
-```
-mvn install:install-file -Dfile=${your-nebula-utils.jar} -DgroupId=com.vesoft -DartifactId=nebula-utils -Dversion={version} -Dpackaging=jar
-```
+However, if it doesn't work in your environment, follow these steps to compile it:
+
+1. Compile the [dynamic link library](https://github.com/vesoft-inc/nebula/tree/master/src/jni).
+2. Install the `jni` jar into your local maven repo.
+
+    ```bash
+    mvn install:install-file -Dfile=${your-nebula-utils.jar} -DgroupId=com.vesoft -DartifactId=nebula-utils -Dversion={version} -Dpackaging=jar
+    ```
 
-See more [Storage Client Examples](https://github.com/vesoft-inc/nebula-java/blob/master/examples/src/main/java/com/vesoft/nebula/examples/) for how to scan edges and vertices from storage directly.
+See more [Storage Client Examples](https://github.com/vesoft-inc/nebula-java/blob/master/examples/src/main/java/com/vesoft/nebula/examples/) about how to scan edges and vertices from storage directly.
 
-### Meta Client
+### Meta client example
 
 See [Meta Client Example](https://github.com/vesoft-inc/nebula-java/blob/master/examples/src/main/java/com/vesoft/nebula/examples/MetaClientExample.java) to access graph schemas.
 
 ## FAQ
 
-Q: Error occurs when building from the source code.
+Q: An error occurs during the process of building from the source code.
 
 ```text
 Failed to execute goal org.apache.maven.plugins:maven-gpg-plugin:1.6:sign (default) on project client: Exit code: 2 -> [Help 1]
 org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal org.apache.maven.plugins:maven-gpg-plugin:1.6:sign (default) on project client: Exit code: 2
 ```
 
-A: This means that you need to have a key to sign the jars
+A: You must generate a key to sign the jars
```

Comment:
or user can skip the gpg:
`mvn clean compile package install -Dgpg.skip -Dmaven.javadoc.skip=true -Dtest.skip=true -Dmaven.test.skip=true`